include/Engine/Input/FirstPersonInputController.h

```diff
@@ -252,4 +253,12 @@ void FirstPersonInputController<EventContext>::AssaultDashCheck(double dt, bool
     m_EventBroker->Publish(e);
 }
 
+template <typename EventContext>
+bool FirstPersonInputController<EventContext>::SniperSprintingCheck() {
+    if (m_SpecialAbilityKeyDown) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
return m_SpecialAbilityKeyDown;
Would be more concise. Also better rename for SniperSprintingCheck perhaps? Like SpecialAbilityKeyDown for example?